### PR TITLE
Disney+: add privacy mode

### PR DESCRIPTION
**Pull request by ThaTiemsz**
Added a privacy mode, also updated the thumbnail.

**Screenshots:**
image
image
image
image
image
image
image

### CODE REVIEW

File: websites/D/Disney+/presence.ts [hunk 1]
```diff
@@ -186,8 +195,12 @@ presence.on("UpdateData", async () => {
       title = titleField?.textContent;
       subtitle = subtitleField?.textContent; // episode or empty if it's a movie
 
-      data.details = title;
-      data.state = subtitle || "Movie";
+      if (privacy) {
+        data.state = subtitle ? strings.watchingSeries : strings.watchingMovie;
+      } else {
```
Comment (ririxidev) [inline websites/D/Disney+/presence.ts:200]:
```suggestion
      if (privacy)
        data.state = subtitle ? strings.watchingSeries : strings.watchingMovie;
      else {
```

File: websites/D/Disney+/presence.ts [hunk 2]
```diff
@@ -83,12 +88,16 @@ presence.on("UpdateData", async () => {
...
-      if (groupWatchId) {
+      if (!privacy && groupWatchId) {
         data.details = `${title} ${subtitle ? `- ${subtitle}` : ""}`;
         data.state = "In a GroupWatch";
       } else {
-        data.details = title;
-        data.state = subtitle || "Movie";
+        if (privacy) {
+          data.state = subtitle ? strings.watchingSeries : strings.watchingMovie;
+        } else {
```
Comment (ririxidev) [inline websites/D/Disney+/presence.ts:97]:
```suggestion
        if (privacy)
          data.state = subtitle ? strings.watchingSeries : strings.watchingMovie;
        else {
```

**Review by ririxidev — CHANGES_REQUESTED** [review]
Heya @ThaTiemsz!
You should check out thing(s) that I requested to change in my review. Please look at them!

Regards,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721.

**Review by ririxidev — APPROVED** [review]
👍

**Review by Bas950 — APPROVED** [review]
LGTM

**Pull request merged**